packages/pluggableWidgets/popup-menu-web/tests/e2e/specs/PopupMenu.spec.ts

```diff
@@ -8,84 +8,92 @@ describe("Popup-menu-web", () => {
     describe("using basic option", () => {
         it("compares with a screenshot baseline and checks if popupmenu is rendered in the top left position", () => {
             browser.setWindowRect(0, 0, 1200, 900);
+            const elem = $(".sprintrFeedback__sidebar");
```

Comment:
Make sure to rebase from master when #404 is merged to master and check whether no changes are left on this file (elem is renamed to sprintrFeedback for instance).